scripts/build_apple_frameworks.sh

```diff
@@ -34,6 +34,29 @@ libextension_module.a,\
 libextension_tensor.a,\
 :${FRAMEWORK_EXECUTORCH_HEADERS_DIR}:${FRAMEWORK_EXECUTORCH_MODULE_NAME}"
 
+FRAMEWORK_EXECUTORCH_LLM_NAME="executorch_llm"
+FRAMEWORK_EXECUTORCH_LLM_MODULE_NAME="ExecuTorchLLM"
+FRAMEWORK_EXECUTORCH_LLM_HEADERS_DIR="${FRAMEWORK_EXECUTORCH_LLM_NAME}_include"
+FRAMEWORK_EXECUTORCH_LLM_HEADERS_PATH="${OUTPUT_DIR}/${FRAMEWORK_EXECUTORCH_LLM_HEADERS_DIR}"
+FRAMEWORK_EXECUTORCH_LLM="${FRAMEWORK_EXECUTORCH_LLM_NAME}:\
+libabsl_hash.a,\
+libabsl_int128.a,\
+libabsl_log_initialize.a,\
+libabsl_log_internal_format.a,\
+libabsl_log_internal_log_sink_set.a,\
+libabsl_log_internal_message.a,\
+libabsl_spinlock_wait.a,\
+libabsl_str_format_internal.a,\
+libabsl_strings.a,\
+libabsl_strings_internal.a,\
+libabsl_time.a,\
+libabsl_time_zone.a,\
```

Comment:
wow

Reply:
I'm wondering how many of them are actually used. I think we only uses `libabsl_strings.a` but not sure if they are inter-dependent.

Reply:
And i'm afraid that's not the full list yet :)